### PR TITLE
Add stimpack to ecosystem section

### PR DESCRIPTION
**Pull request by exterm**
## What are you trying to accomplish?
Make `stimpack` more discoverable for people that want to use packwerk similarly to how Gusto is using it

## What approach did you choose and why?
I think a pointer makes sense, I don't want to make it a recommendation as that would suggest `stimpack`'s patterns are the one recommended path.

## What should reviewers focus on?

Any drawbacks to adding this shoutout?

## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Non-breaking change (a change that doesn't alter functionality - i.e., code refactor, configs, etc.)

## Checklist

- [x] It is safe to rollback this change.

### CODE REVIEW

**Comment by exterm** [comment]
@AlexEvanczuk in reference to https://github.com/Shopify/packwerk/issues/177

File: README.md [hunk 1]
```diff
@@ -58,10 +58,11 @@ Read [USAGE.md](USAGE.md) for usage once Packwerk is installed on your project.
 
 ## Ecosystem
 
-You can use these third party tools to enhance your packwerk experience:
+Various third parties have built tooling on top of packwerk. Here's a selection of some that might prove useful:
 
 - https://github.com/bellroy/graphwerk draws a graph of your package dependencies
 - https://github.com/Gusto/packwerk-vscode integrates packwerk into Visual Studio Code so you can see violations right in your editor
+- https://github.com/Gusto/stimpack is a set of conventions and tooling that makes packwerk packages more like Rails engines.
```
Comment (AlexEvanczuk) [inline README.md:65]: Thank you for the inclusion here @exterm . Actually I think we want to de-emphasize rails engine similarity as a selling point.

Making a PR to this end in `stimpack`: https://github.com/Gusto/stimpack/pull/3

Our blurb says:
> `stimpack` establishes and implements a set of conventions for splitting up large monoliths built on top of [`packwerk`](https://github.com/Shopify/packwerk). With `stimpack`, new packages' autoload paths are automatically added to Rails, so your code will immediately become usable and loadable without additional configuration.

Can we use this as the blurb? We could also take out the second sentence if you feel like we want something more concise.

Comment (exterm) [reply README.md:65]: I'd love to be a little more specific than just "a set of conventions" and I don't think I'm familiar enough with stimpack to summarize what those conventions actually are 🤔 

Other than that, I am on board 😄

Comment (AlexEvanczuk) [reply README.md:65]: Yep, that's the basic idea.

Comment (exterm) [reply README.md:65]: `- https://github.com/Gusto/stimpack sets up autoloading and more for a flat list of packages.`

WDYT?

File: README.md [hunk 2]
```diff
@@ -58,10 +58,11 @@ Read [USAGE.md](USAGE.md) for usage once Packwerk is installed on your project.
 
 ## Ecosystem
 
-You can use these third party tools to enhance your packwerk experience:
+Various third parties have built tooling on top of packwerk. Here's a selection of some that might prove useful:
 
 - https://github.com/bellroy/graphwerk draws a graph of your package dependencies
 - https://github.com/Gusto/packwerk-vscode integrates packwerk into Visual Studio Code so you can see violations right in your editor
+- https://github.com/Gusto/stimpack sets up Rails autoloading, as well as `rspec` and `FactoryBot` integration, for packages arranged in a flat list.
```
[the remark this answers is not in the document]
Comment (AlexEvanczuk) [reply README.md:65]: This looks great thank you 🙏

[the remark this answers is not in the document]
Comment (rafaelfranca) [reply README.md:65]: Curious, why setting Rails autoloading with a library if Rails engines already do that? Isn't that trying to reimplement the framework?

[the remark this answers is not in the document]
Comment (exterm) [reply README.md:65]: Added a [cautionary message](https://github.com/Shopify/packwerk/pull/187/commits/2f4ec0e4e2524dbb54e40b8250c3b58ffd84d06d) while we're waiting for a conclusion to this discussion. Maybe @ngan has a good answer to Rafael's question while we're waiting for Alex to return from vacation?

**Pull request merged**